### ISSUE TITLE
Whatsapp web: cannot paste screen capture from the clipboard

### ISSUE THREAD

**Issue by Kazhuu** [fanglingsu/vimb#650]
###Steps to reproduce
Go to whatsapp web app and login. Then have screen capture on the clipboard taken with flameshot for example. Try to paste it to the chat.
### Expected behaviour
Window should come up asking do you want to paste the image.
### Actual behaviour
No popup comes up.

I'm using the latest trunk build. Text can be pasted to the chat by pressing C-V and then C-V again. But same doesn't happen with image for some reason. Any clue what prevents this? I checked console for JS errors too but none came up.

**Comment by fanglingsu** [comment]
@Kazhuu I can reproduce this by putting an image on clipboard `xclip -selection clipboard -t image/png -i test.png` and pasting it into https://www.textbox.io/. This seems not to work with webkit, but I'm not sure how this feature is called.

**Comment by Kazhuu** [comment]
Yeah I get the same results as you above. I'm not so familiar with webkit or this feature what it's called. Hopefully someone can enlighten us with their knowledge what is the problem. I might look into this if I have time in the future. Thanks for the help!
